packages/components/src/Disclosure/tests/chromatic/Disclosure.stories.tsx

```diff
@@ -0,0 +1,194 @@
+        </DisclosurePanel>
+    </Disclosure>
+);
+
+export const DefaultStates = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        const triggers = canvas.getAllByRole("button");
+
+        triggers.forEach(trigger => {
+            if (trigger.getAttribute("disabled") !== "") {
+                const disclosureElem = trigger.closest(".hop-Disclosure");
+
+                if (disclosureElem?.getAttribute("data-chromatic-force-focus")) {
+                    trigger?.setAttribute("data-focus-visible", "true");
+                    disclosureElem?.removeAttribute("data-chromatic-force-focus");
+                }
+
+                if (disclosureElem?.getAttribute("data-chromatic-force-press")) {
+                    trigger?.setAttribute("data-pressed", "true");
+                    disclosureElem?.removeAttribute("data-chromatic-force-press");
+                }
+
+                if (disclosureElem?.getAttribute("data-chromatic-force-hover")) {
+                    trigger.setAttribute("data-hovered", "true");
+                    disclosureElem?.removeAttribute("data-chromatic-force-hover");
+                }
+            }
+        });
+    },
+    render: args => (
+        <Stack>
+            <h1>Default</h1>
+            <StateTemplate {...args} />
+            <h1>Disabled</h1>
+            <StateTemplate {...args} isDisabled />
+            <h1>Focus Visible</h1>
+            <StateTemplate {...args} data-chromatic-force-focus />
+            <h1>Hovered</h1>
+            <StateTemplate {...args} data-chromatic-force-hover />
+            <h1>Pressed</h1>
+            <StateTemplate {...args} data-chromatic-force-press />
+            <h1>Focus Visible & Disabled</h1>
+            <StateTemplate {...args} data-chromatic-force-focus isDisabled />
+        </Stack>
+    )
+} satisfies Story;
+
+export const InlineStates = {
+    ...DefaultStates,
+    args: {
+        variant: "inline"
+    }
+} satisfies Story;
+
+export const Zoom = {
+    render: args => (
+        <Stack>
+            <Disclosure {...args} className="zoom-in">
+                <DisclosureHeader prefix={<SparklesIcon />} level={2}>
+                    <Text>Shipping, Delivery Times, and Easy Returns Policy Overview</Text>
+                    <Text slot="description">Explore our comprehensive shipping options, estimated delivery times for various regions, and our simple, customer-friendly returns process to make sure you feel comfortable with every purchase.</Text>
+                </DisclosureHeader>
+                <DisclosurePanel>
+                We offer free standard shipping on all orders over $50. Orders are typically processed within 1-2 business days, and delivery times vary based on your location. Expedited shipping options are available for an additional fee.
```

Comment:
It's just a test but shouldn't panels if no text wrapper is present default to a paragraph element?

Reply:
not sure I know what you mean? are you saying the user should add the p tags? they can add whatever they want..

Reply:
this renders in no <p> or <span>

Reply:
oh I see what you're saying, I guess it may make sense to do this.